code/pizzabot.js

```diff
@@ -1,7 +1,95 @@
+
+// *** declairing variables
 const vegetarian = "Vegetarian Pizza"
 const hawaiian = "Hawaiian Pizza"
 const pepperoni = "Pepperoni Pizza"
+const pizzaPrice = 80;
+
+const pizzaVegImg = document.getElementById("veg");
+const pizzaHawImg = document.getElementById("haw");
+const pizzaPepImg = document.getElementById("pep");
+
+
+// !! HIDING THIS PART SINCE IT ONLY WAS PART OF ITERATION 1-7. !!
+/*** Function 1 - Order Name 
+    const validateOrderName = (order) => {
+        if (order === vegetarian || order === hawaiian || order === pepperoni) {
+          return true;
+        } else {
+          return false;
+        }
+      };
+***/
+
+// *** Function 2 - Total Cost
+const calculateTotalCost = (quantity, price) => {
+  return quantity * price;
+}
+
+// *** Function 3 - Cooking Time
+const calculateCookingTime = (time) => {
+  if (time <= 2) {
+    return 10;
+  } else if (time >= 3 && time <=5) {
+    return 15;
+  } else {
+    return 20;
+  }
+}
+
+// *** Function - Submit form
+
+const handleSubmit = (event) => {
+  event.preventDefault();
+  let userName = document.forms.pizzaForm.userName.value;
+  // makes first letter to uppercase and the rest of the mane to lowercase
+  userName = userName.charAt(0).toUpperCase() + userName.slice(1).toLowerCase();
+
+  const pizzaType = document.forms.pizzaForm.elements.pizza.value;
+  const orderQuantity = document.forms.pizzaForm.numberOfPizzas.value;
+  
+    const orderTotal = calculateTotalCost(orderQuantity, pizzaPrice);
+    const cookingTime = calculateCookingTime(orderQuantity);
+
+    // 'ternary operator' (puts the 's for plural number of pizzas)
+    const plural = orderQuantity > 1 ? "'s" : "";
+
+    document.getElementById(
+      "orderSummery"
+    ).innerHTML = `Hello ${userName}! I'll get started on your ${orderQuantity} ${
+      pizzaType + plural
+    } right away. It will cost ${orderTotal} kr and will take ${cookingTime} minutes.`;
+    
+    if (pizzaType === "Vegetarian Pizza") {
+      pizzaVegImg.setAttribute ("style", "display:block;");
+    } else if (pizzaType === "Hawaiian Pizza") {
+      pizzaHawImg.setAttribute ("style", "display:block;")
+    } else if (pizzaType === "Pepperoni Pizza") {
+      pizzaPepImg.setAttribute ("style", "display:block;")
```

Comment:
That's great you know that there is option to manipulate styles via JS, but this antipattern - what you can do instead is to manipulate `classList` property and add or remove classes.
Main reason for that is, as convention says - only code inside CSS files should be responsible for handling styles to avoid chaos in code